### PR TITLE
{vis}[intel/2017b] basemap v1.0.7, GEOS v3.6.2, Pillow v5.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
{vis}[intel/2017b] basemap v1.0.7

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2433.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b53cca01c4a6da899a82e322a4ffa200 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2045.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/548a73b469cd2e9c20600d6c9bde1669 for a full test report.

File: easybuild/easyconfigs/b/basemap/basemap-1.0.7-intel-2017b-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org/basemap/'
+description = """The matplotlib basemap toolkit is a library for plotting 2D data on maps in Python"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['https://downloads.sourceforge.net/project/matplotlib/matplotlib-toolkits/basemap-%(version)s']
+
```
Comment (vanzod) [inline easybuild/easyconfigs/b/basemap/basemap-1.0.7-intel-2017b-Python-3.6.3.eb]: Missing checksum

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**